orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -49,6 +49,30 @@ def __init__(self, state, output_info: List[Tuple[torch.Size, torch.device, torc
         self.output_info = output_info
 
 
```

Comment:
add comment for why we need this clas and how it works.?

Reply:
done, thx